tests/integration/components/select/country-select-test.js

```diff
@@ -0,0 +1,31 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import selectCountry from '../../../pages/components/select/country-select';
+import PageObject from 'ember-cli-page-object';
+
+let page = PageObject.create(selectCountry);
+
+moduleForComponent('select/country-select', 'Integration | Component | select/country select', {
+  integration: true,
+  beforeEach() {
+    page.setContext(this);
+  },
+  afterEach() {
+    page.setContext(this);
+  }
+});
+
+test('it sets the correct country', function(assert) {
+  assert.expect(1);
+
+  this.set('country', 'US');
+
+  page.render(hbs`{{select/country-select country=country}}`);
+
+  page.country.fillIn('US');
+
+  let country = this.get('country');
```

Comment:
Should `birth-date` also use this? I agree this is better I just didn't do it initially since i didn't see any other tests using it